message_sender/views.py

```diff
@@ -98,6 +98,14 @@ def get_serializer_class(self):
         return InboundSerializer
 
     def create(self, request, *args, **kwargs):
+        if ("duration" in
+                request.data.get("helper_metadata", {}).get("voice", {})):
+            fire_metric.apply_async(kwargs={
+                "metric_name": "vumimessage.obd.registration.length",
```

Comment:
I don't think that incoming registration calls get to the message sender. That will have to be done in the jsbox app, that's the only thing that sees the registration calls.